jenkins/gce-custom-images/prepare_custom_leonardo_gce_image.sh

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env bash
```

Comment:
It would be good to bake the "hardening" stuff that Albano linked into this script

Reply:
Right. I wanted to see what that hardening stuff entails and then decide whether it makes sense to subsume it into this script, call it from this script or call it from Jenkins as a subsequent step. It might be good to keep the hardening script (or whatever it turns out to be) a separate thing if it applies to other image creations in the future as well.